examples/linkingalps/LA_5T_TR_Request .xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../OJP.xsd">
+	<OJPRequest>
+		<siri:ServiceRequest>
+			<siri:RequestTimestamp>2023-06-13T17:53:00.000Z</siri:RequestTimestamp>
+			<siri:RequestorRef>SBB-SoapUI-Tests</siri:RequestorRef>
+			<OJPTripRequest>
+				<siri:RequestTimestamp>2023-06-13T17:53:00.000Z</siri:RequestTimestamp>
+				<Origin>
+					<PlaceRef>
+						<StopPlaceRef>23:1917</StopPlaceRef>
+						<Name>
+							<Text>Origin</Text>
+						</Name>
+					</PlaceRef>
+					<DepArrTime>2023-06-13T17:53:00.000Z</DepArrTime>
+				</Origin>
+				<Destination>
+					<PlaceRef>
+						<StopPlaceRef>gtt:1992</StopPlaceRef>
+						<Name>
+							<Text>Desitination</Text>
```

Comment:
Destination

Reply:
Even worse: this file has a space in it.

Reply:
I fixed that in https://github.com/VDVde/OJP/pull/374

Reply:
I wanted to approve it, but it somehow didn't work...